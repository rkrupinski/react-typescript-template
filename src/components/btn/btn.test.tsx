import * as React from 'react';
import * as renderer from 'react-test-renderer';

import Btn from '.';

it('Renders btn', function () {
  ['Lorem', 'Ipsum', 'ðŸ’©'].forEach((text) => {
    const component = renderer.create(
      <Btn>{text}</Btn>
    );

    expect(component).toMatchSnapshot();
  });
});
