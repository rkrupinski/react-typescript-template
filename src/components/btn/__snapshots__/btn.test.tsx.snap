// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders btn 1`] = `
<button
  className="cHlpJQ"
>
  Lorem
</button>
`;

exports[`Renders btn 2`] = `
<button
  className="cHlpJQ"
>
  Ipsum
</button>
`;

exports[`Renders btn 3`] = `
<button
  className="cHlpJQ"
>
  ðŸ’©
</button>
`;
